llvm/lib/Target/AArch64/AArch64InstrInfo.td

```diff
@@ -2568,12 +2580,59 @@ defm CASPA  : CompareAndSwapPair<1, 0, "a">;
 defm CASPL  : CompareAndSwapPair<0, 1, "l">;
 defm CASPAL : CompareAndSwapPair<1, 1, "al">;
 
+// v9.6-a atomic CAST
```

Comment:
I believe that these CAS and CASP have alias too

Reply:
Done